src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {

```

Comment:
The class IsoscelesTrapezoid should not begin with an empty line. Please remove the empty line at the beginning of the class definition.